<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElbowsDockWidget</class>
 <widget class="QWidget" name="ElbowsDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>381</width>
     <height>661</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>361</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>756</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {
    min-width: 55px;  /* 设置标签宽度，根据实际内容调大 */
    min-height: 30px;  /* 设置标签高度 */
    padding: 2px;       /* 内边距，避免文字贴边 */
}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="iconSize">
    <size>
     <width>120</width>
     <height>16</height>
    </size>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <property name="minimumSize">
     <size>
      <width>357</width>
      <height>578</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::tab {
    min-width: 100px;
    min-height: 30px;
}
</string>
    </property>
    <attribute name="title">
     <string>弯管参数</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>481</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>管体外半径：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_Rout">
        <property name="text">
         <string>32</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>管体内半径：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_Rin">
        <property name="text">
         <string>30</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>管体长度：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_Length">
        <property name="text">
         <string>500</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>旋转套筒厚度：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_SleeveThickness">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>固定套筒长度：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_SleeveLength">
        <property name="text">
         <string>200</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>固定套筒厚度：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>旋转套筒长度：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_SleeveThickness1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_SleeveLength1">
        <property name="text">
         <string>20</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>旋转套筒位置：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_RotaryPos">
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>固定套筒位置：</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_FixedPos">
        <property name="text">
         <string>250</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>半圆弧套半径：</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEdit_ArcR">
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>半圆弧套厚度：</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEdit_ArcThickness">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>半圆弧弧度：</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="lineEdit_ArcAngle">
        <property name="text">
         <string>4.7124</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QPushButton" name="pushButton_confirm">
        <property name="styleSheet">
         <string notr="true">QPushButton {
        background-color: rgb(119, 118, 123); 
        color: white; 
        border: none; 
        }
        QPushButton:hover {
        background-color: rgb(94, 92, 100); 
        }</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>材料属性</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>管体：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_tubeMaterial">
        <property name="styleSheet">
         <string notr="true">QComboBox QAbstractItemView {
    padding: 4px;
}

</string>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>304不锈钢</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>铝合金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>钛合金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>黄铜</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>黄金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>塑料</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>刚体</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>旋转套筒：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_rotaryMaterial">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>304不锈钢</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>铝合金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>钛合金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>黄铜</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>黄金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>塑料</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>刚体</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>固定套筒：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_fixedMaterial">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>304不锈钢</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>铝合金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>钛合金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>黄铜</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>黄金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>塑料</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>刚体</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>半圆弧套筒：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_arcMaterial">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>304不锈钢</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>铝合金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>钛合金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>黄铜</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>黄金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>塑料</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>刚体</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>工艺求解</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>网格大小：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-left: 3px;</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_generateMesh">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
        background-color: rgb(119, 118, 123); 
        color: white; 
        border: none; 
        }
        QPushButton:hover {
        background-color: rgb(94, 92, 100); 
        }</string>
        </property>
        <property name="text">
         <string>网格划分</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>结果展示</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_newFile">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../MainWindow.qrc">
     <normaloff>:/main_wind/figure/main_wind/new.png</normaloff>:/main_wind/figure/main_wind/new.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_openFile">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../MainWindow.qrc">
     <normaloff>:/main_wind/figure/main_wind/open.png</normaloff>:/main_wind/figure/main_wind/open.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveFile">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../MainWindow.qrc">
     <normaloff>:/main_wind/figure/main_wind/save.png</normaloff>:/main_wind/figure/main_wind/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
